tests/sample-repo/.snyk

```diff
@@ -1,5 +1,5 @@
 exclude:
   global:
-    - exclude/excluded-folder/**
```

Comment:
IMO we should also have a test case for paths without the leading slash - i.e. between excluding just the root `exclude/excluded-folder` and excluding any nested `exclude/excluded-folder`.